### PR TITLE
[WNMGDS-2224] Document Preact and web components

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

https://jira.cms.gov/browse/WNMGDS-2224

- Document the additional component-framework offerings and import strategies in our developer docs

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [x] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [x] Selected appropriate `Impacts`, multiple can be selected.

### If this is a change to documentation:

- [ ] Checked for spelling and grammatical errors

### CODE REVIEW

File: packages/docs/content/getting-started/for-developers.mdx [hunk 1]
```diff
@@ -377,6 +377,36 @@ import { Button, TextField } from 'ds-medicare-gov';
 
 </ThemeContent>
 
+While you can actually use Preact inside code that was written for React—and [the preact-react-app example](https://github.com/CMSgov/design-system/tree/main/examples/preact-react-app) shows you how—we've also provided an import path for you to use the Preact version of our components directly. To import the Preact versions, add `/preact` to the end of your import path like this:
```
Comment (forrestbaer) [inline packages/docs/content/getting-started/for-developers.mdx:380]: `-and` seems like a typo?

Comment (pwolfert) [reply packages/docs/content/getting-started/for-developers.mdx:380]: In monospace you can't tell that it's an emdash, but it should look right on the doc site. I'm open to rewriting this sentence to be simpler

Comment (forrestbaer) [reply packages/docs/content/getting-started/for-developers.mdx:380]: `While it is possible to use Preact code inside React code` ?

Comment (pwolfert) [reply packages/docs/content/getting-started/for-developers.mdx:380]: oh yeah, that's worded funny. inside a React project. whole thing maybe needs rewording. I had a hard time explaining it succinctly

Comment (pwolfert) [reply packages/docs/content/getting-started/for-developers.mdx:380]: total rewrite coming

File: packages/docs/content/getting-started/for-developers.mdx [hunk 2]
```diff
@@ -377,6 +377,44 @@ import { Button, TextField } from 'ds-medicare-gov';
...
+
+```js
+import { Button, TextField } from 'ds-medicare-gov/preact';
+```
+
+</ThemeContent>
+
+### Using web components
+
+We're also starting to offer [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) as a framework-agnostic alternative to our React or Preact components. We will be working on expanding this library over time, but at a moment only a subset of our JavaScript components are represented in this library. The easiest way to consume these components is by [loading them from the CDN](https://github.com/CMSgov/design-system/tree/main/examples/cdn-web-components); however, they can also be installed via npm and imported by appending `/web-components` to your import path.
```
Comment (forrestbaer) [inline packages/docs/content/getting-started/for-developers.mdx:416]: I think this should be past tense as by the time someone is reading it this is already true. So `We've also begun (or started) offering`

**Review by forrestbaer — APPROVED** [review]
Looks good just one more comment, I hate being too pedantic about grammar! :)

**Pull request merged**